packages/paste-libraries/uid/src/index.tsx

```diff
@@ -1 +1,17 @@
-export {UIDFork, useUID, useUIDSeed, uid} from 'react-uid';
+import * as React from 'react';
+import {useUID as useUIDLegacy, useUIDSeed as useUIDSeedLegacy} from 'react-uid';
+
+let useUID;
+let useUIDSeed;
+
+if (React.useId) {
+  useUID = React.useId;
+  useUIDSeed = useUIDSeedLegacy;
```

Comment:
Does this work as something like this by any chance?
```
useUIDSeed = () => React.useId()
```

Reply:
You're right just added it!